talent/job-descriptions/ml-engineering-section.md

```diff
@@ -1,10 +1,13 @@
 ### Engineering at source{d}
 
-- The Machine Learning team is focused on providing intelligent insights into nearly all the world's open source code. This includes typical big data analysis as well as solving sophisticated Machine Learning problems. The former is conducted on Apache Spark clusters with up to 1,000 nodes, the latter leverages extensive GPGPU acceleration on the custom hardware and Deep Learning techniques, standing on top of source{d}'s original open source projects and Tensorflow. Often, arising ML problems are unsupervised and require much research in the unusual domain of ML on source code (MLoSC).
+The machine learning team is focused on providing intelligent insights into nearly all the world's open source code. This includes typical big data analysis as well as solving sophisticated machine learning problems. The former is conducted on Apache Spark clusters, the latter leverages extensive GPGPU acceleration on custom hardware using deep learning techniques. Often ML problems we face are unsupervised and require novel research in the domain of ML on source code (MLoSC).
```

Comment:
- s/machine learning/Machine Learning/g
- s/big data/Big Data/g
- s/ML on source code/Machine Learning on Source Code/g

I'd drop the "intelligent" and simply provide insights
Add a link to the definition of GPGPU